content/contributing/project-information/release-process.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Release Process"
+insert_anchor_links = "right"
+[extra]
+weight = 3
++++
+
+Bevy uses three-months-long development cycles, delimited by a weeks-long rolling release process.
```

Comment:
```suggestion
Bevy uses a three-months-long development cycles, delimited by a weeks-long rolling release process.
```

Reply:
I don't agree with this suggestion!

Reply:
Now that I look at it again, I missed the s at the end of cycles. My bad.